README.md

```diff
@@ -117,6 +117,134 @@ class Test {
 ```
 
 
-## Future work
-  - Converters for `java.util.stream`
-  - [`Spliterator`](https://docs.oracle.com/javase/8/docs/api/java/util/Spliterator.html)s for Scala collections
+## Converters from Scala collections to Java 8 Streams
+
+Scala collections gain `seqStream` and `parStream` as extension methods that produce a Java 8 Stream
+running sequentially or in parallel, respectively.  These are automatically specialized to a primitive
+type if possible.  For instance, `List(1,2).seqStream` produces an `IntStream`.  Maps additionally have
+`seqKeyStream`, `seqValueStream`, `parKeyStream`, and `parValueStream` methods.
+
+Scala collections also gain `accumulate` and `stepper` methods that produce utility collections that
```

Comment:
Adriaan asked me to write a framework for exposing more collections internals so it was easier to do things like write parallelizing code.  It seemed that having a Stepper already there would be a benefit.  But I agree, it could probably be hidden as an implementation detail (at least with `private[scala.compat]`).

I'm not so sure it makes sense to hide Accumulator.  If you want to safely collect intermediate results of unknown size, it is the thing to use.  There are no equivalents.